pkg/indicator/v2/breakaway.go

```diff
@@ -0,0 +1,65 @@
+package indicatorv2
+
+import (
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+type BreakAwayStream struct {
+	*types.Float64Series
+
+	window int
+}
+
+func BreakAway(source KLineSubscription) *BreakAwayStream {
+	s := &BreakAwayStream{
+		Float64Series: types.NewFloat64Series(),
+		window:        5,
+	}
+
+	source.AddSubscriber(func(kLine types.KLine) {
+		var (
+			i      = source.Length()
+			output = Neutral
+		)
+		if i < s.window {
+			s.PushAndEmit(output)
+			return
+		}
+		var (
+			five  = source.Last(4)
+			four  = source.Last(3)
+			three = source.Last(2)
+			two   = source.Last(1)
+			one   = source.Last(0)
+		)
+
+		if five.Open < five.Close {
```

Comment:
five.Open is a fixedpoint.Value, you need to call .Compare() because we have two implementation for fixedpoint.Value

Reply:
ouch. so this whole fixedpoint.Value thing is a bit cumbersome to use imo. I know golang does not support decimal natively but the approach to fixedpoint is also not consistent throughout the code. Ever thought of a refactor of this?
I tried with a fork of a 128decimal fixed point decimal lib and ported all the missing util functions present in fixedpoint. 
[here](https://github.com/algo-boyz/decimal128) compiled but quite a few tests still failing for subtle differences. So I parked it for now. Computation time is faster tho but in general the state of play in go with fixedpoint precision is a trouble and it shows in the code :(

Reply:
and weirdly this gives the right result without using Compare or how would the test pass?

Reply:
Can you point out the inconsistent parts?
Usually we convert from fixedpoint to float64 for accounting non-related calculation in order to get speed. High precision calculation in strategy is usually not necessary.

Also fixedpoint is able to support high precision calculation with different compilation flag(recommended for institutional production use). Not using Compare() will end in unexpected result under the high precision fixedpoint.

Reply:
Please also check https://github.com/c9s/bbgo/blob/main/pkg/fixedpoint/dec.go for the high precision implementation (dnum). This one is battle tested.

Reply:
I updated the PR to respect comparators